extensions/sql-migration/src/dialog/tableMigrationSelection/tableMigrationSelectionDialog.ts

```diff
@@ -112,43 +157,144 @@ export class TableMigrationSelectionDialog {
 		}
 	}
 
+	private async _createMigrationContainer(view: azdata.ModelView): Promise<azdata.FlexContainer> {
+		// Schema migration components
+		const schemaMigrationHeader = view.modelBuilder.text()
+			.withProps({
+				value: "Schema migration",
+				requiredIndicator: true,
+				CSSStyles: { ...styles.SECTION_HEADER_CSS, 'margin-top': '4px' }
+			}).component();
+
+		this._schemaMigrationCheckBox = view.modelBuilder.checkBox()
+			.withProps({
+				checked: false,
+				label: "Migrate schema to target",
```

Comment:
same here, needs to be a resource string

Reply:
done.